.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+### Extra Detail
+
+#### Screenshots
+<If bug fix, please upload images of the error to better showcase the problem.>
+
+#### Reasoning
+<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>
+
+### How to Contribute
+- [ ] Comment on this issue if you'd like to work/collaborate on it!
+- [ ] Fork the [repository](https://github.com/github/training-kit).
+- [ ] Clone YOUR forked version of the repository to your command line. `git clone YOURURL`
+- [ ] Make changes.
+- In your command line:
+  - [ ] ADVANCED/OPTIONAL: If you'd like to have a copy of the files to be served from a web server inside of your firewall, start by running `script/package`.
+  - [ ] Run `script/setup`
+  - [ ] Run `script/server`.
```

Comment:
To @brianamarie's point, and speaking from personal experience, I think this step is where most people will struggle. But I also don't think we should blow up the issue with a thousand troubleshooting steps. Instead, I think we should 👀 look for and leverage existing 💎 ruby installation instructions and troubleshooting.